docker/alpine/Dockerfile

```
@@ -91,74 +91,69 @@ RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        -o cpack_name=dist \
        -o cpack_generator=TGZ \
        -pr "$CONAN_PROFILE" \
-       ${WITH_TESTS_ARG}
+       "${WITH_TESTS_ARG}" \
+  && tar xf dist.tar.gz \
+  && rm dist.tar.gz
 
 # ---------------------------------------------------------------------
 # release
 # ---------------------------------------------------------------------
 
 FROM os AS release
 
-# TODO: How about shortening the description to MTConnect Agent or at least MTConnect C++ Agent?
 LABEL author='mtconnect' description='MTConnect C++ Agent'
 
-# install ruby for simulator
-RUN  apk update \
-  && apk add \
-       ruby \
-       libstdc++ \
-       libgcc
-
 # change to a new non-root user for better security.
 # this also adds the user to a group with the same name.
 # --create-home creates a home folder, ie /home/<username>
 ARG UID=1000
 ARG GID=1000
 
-RUN  addgroup \
+# install dependencies
+RUN  apk update \
+  && apk add \
```

Consider replacing this with the following:

```suggestion
RUN apk --update add \
```

---

changed